connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -1118,4 +1118,18 @@ private[v2] trait V2JDBCTest extends SharedSparkSession with DockerIntegrationFu
       testBinaryLiteral("<=>", greaterThanBinary, 0)
     }
   }
+
+  test("SPARK-52262: FAILED_JDBC.TABLE_EXISTS not thrown on connecttion error") {
```

Comment:
nit: double t
```suggestion
  test("SPARK-52262: FAILED_JDBC.TABLE_EXISTS not thrown on connection error") {
```
Also, I would recommend naming function in a way to describe new positive behaviour than to descirbe it to old negative behaviour

Reply:
I would leave like this, since current behavior is still not idea, so this is easier to realize that it's ok to change this test if analyzer were to be changed.

Reply:
Agreed, what is current error condition?

Reply:
Fixed the test now, it was issue that catalog is already initialized when running whole suite.

Reply:
`TABLE_OR_VIEW_NOT_FOUND`